PREREQUISITES.md

```diff
@@ -0,0 +1,309 @@
+# Prerequisites and Setup
+
+> [!Important]
+> The course is dense.
+>
+> You won't have time to install and configure everything during on D-day.
+>
+> You won't be able to follow the course if you don't have everything installed and working.
+>
+> 📣 **Please make sure you have everything installed and working before the course starts.** 📣
+>
+> You know your school WiFi better than we do, don't gamble on it.
+
+> [!Important]
+> We will be using Docker for this course, which can use up a lot of your disk.
+> Make sure to have at least 10Gb available in your disk before starting this course. Otherwise you risk running into some obscure errors.
+
+> [!Note]
+> Each section has a **Check your Installation** section.
+> Please make sure you can run the commands in that section before moving on to the next section.
```

Comment:
Peut-être ajouter à la fin : "If any trouble during one of this step, please send an email to...."